### PR TITLE
Small Quality of Life changes

### PR DESCRIPTION
**Pull request by lucca65**
## What issue does this PR close
Closes N/A

## Changes Proposed ( a list of new changes introduced by this PR)
This PR contains a couple of small fixes that appeared during some meetings about features. Small Quality of Life improvements:
- After the user did a claim, change the message to be less robotic `Congrats! You are close to earn your {{symbolCode}}. We are reviewing your action`
- Change a text that was grey to be white on the the Action Card (Objectives page) 
- Change redirect after editing an Action, it was redirecting to the community page, now it takes the user back to Objectives page
- Change how we do a selections for symbol, now it supports any string to be converted to a symbol. Also updated usages of the selection set
- Fix display of symbols on the notification page
- Add our default header to the notification page
- Remove "New Objective" link from the community page, it was causing confusion to new admins
- Make the community page header to be smaller so it works better on mobile, as its used by users to do claims

### CODE REVIEW

File: src/elm/Page/Community.elm
Comment (amiskov) [inline src/elm/Page/Community.elm:145]: We don't need `editStatus` anymore, but I already removed it in `feature/claimer-flow-with-photo` so we can leave it here for now. It will be removed alongside with `EditStatus` type while merging `new_claims`.

**Review by amiskov — APPROVED** [review]
Looks good!

Probably, there will be some conflicts while merging with `new_claims` but I will handle them, not a big deal.

**Comment by lucca65** [comment]
let's wait to merge this PR after we merged claims then! claims are far more relevant

**Comment by amiskov** [comment]
> let's wait to merge this PR after we merged claims then! claims are far more relevant

@lucca65 May I ask you to merge this PR first? I think it will be much simpler to merge these changes into `new_claims` branch the do the opposite.

**Comment by amiskov** [comment]
@lucca65 Well, since I have this superpower I just merged this PR by myself 🙂

**Comment by lucca65** [comment]
ok! hehehe

**Pull request merged**
